javascript/src/util.ts

```diff
@@ -1,8 +1,3 @@
 export interface PostOptions {
     idempotencyKey?: string;
 }
-
-export interface ListOptions {
-    iterator?: string | null;
-    limit?: number;
-}
```

Comment:
Are we exporting this from the package? If so, we need to document that this will be a breaking change

Reply:
No, it was only exported for use in other modules.